### PR TITLE
PLENTY-146 Convert existing styles to SCSS

### PR DESCRIPTION
**Pull request by darius-adeoweb**

### CODE REVIEW

**Comment by darius-adeoweb** [comment]
There was no predefined variables for colors in plugin-ceres. Maybe we should create a separate Findologic variables file and include it in findologic.scss? Also media break pixel size could also be set as a variable there.

File: resources/scss/findologic.scss [hunk 1]
```diff
@@ -1,3 +1,7 @@
 @import '../../vendor/plentymarkets/plugin-ceres/resources/scss/variables';
 
-@import 'views/ItemList/Components/Filter/item-filter-list';
\ No newline at end of file
+@import 'views/ItemList/Components/Filter/item-filter-list';
+@import 'views/ItemList/Components/item-color-tiles';
+@import 'views/ItemList/Components/item-range-slider';
+@import 'views/ItemList/Components/item-search';
+@import 'views/Category/Item/Partials/promotion';
```
Comment (TheKeymaster) [inline resources/scss/findologic.scss]: Add newlines at the end of each newly added file.

File: resources/scss/views/ItemList/Components/item-color-tiles.scss [hunk 2]
```diff
@@ -0,0 +1,47 @@
+.fl-item-color-tiles-container {
+    .fl-item-color-tiles-list {
+        padding-left: 0;
+
+        .fl-item-color-tiles-list-item {
+            margin: 10px 10px 10px 0;
+            display: inline-block;
+
+            .fl-color-tile-label {
+                border: 1px solid #ccc;
```
Comment (TheKeymaster) [inline resources/scss/views/ItemList/Components/item-color-tiles.scss]: Use `$gray-lighter2`

File: resources/scss/views/ItemList/Components/item-color-tiles.scss [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+                position: relative;
+                cursor: pointer;
+
+                svg {
+                    width: 24px;
+                    height: 24px;
+                    margin: 3px;
+
+                    path {
+                        stroke: black;
```
Comment (TheKeymaster) [inline resources/scss/views/ItemList/Components/item-color-tiles.scss]: Use `$black`

File: resources/scss/views/ItemList/Components/item-color-tiles.scss [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+                        fill: white;
```
Comment (TheKeymaster) [inline resources/scss/views/ItemList/Components/item-color-tiles.scss]: Use `$white`

File: resources/scss/views/ItemList/Components/item-color-tiles.scss [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+                        fill: $white;
+                    }
+                }
+                .fl-color-tile-background {
+                    width: 30px;
+                    height: 30px;
+                    text-align: center;
+
+                    &:hover {
+                        border: 1px solid #ddd;
```
Comment (TheKeymaster) [inline resources/scss/views/ItemList/Components/item-color-tiles.scss]: I have overseen this in the last review. Please use `$custom-file-border-color`.

Comment (darius-adeoweb) [reply resources/scss/views/ItemList/Components/item-color-tiles.scss]: I would recommend to create a new Findologic plugin variables file and define this variable with name in this plugin context, something like $color-tiles-border-color.

Comment (TheKeymaster) [reply resources/scss/views/ItemList/Components/item-color-tiles.scss]: We won't need this right now.

**Pull request merged**